pkg/plugin/aggregation/handler.go

```diff
@@ -35,6 +36,10 @@ const (
 	resultsByNode = "/api/v1/results/by-node/{node}/{plugin}"
 	// resultsGlobal is the path for global (non node-specific) results to be PUT
 	resultsGlobal = "/api/v1/results/global/{plugin}"
+
+	// defaultFilename is the name given to the file if no filename is given in the
+	// content-disposition header
+	defaultFilename = "result"
```

Comment:
It used to be that when single files were uploaded, the node name was used as the result file name. Now, since we are keeping the node name as the folder, we needed to come up with a default name.

Not married to this so if you have a preference let me know.